cypress/e2e/tests/migration/dynamic-report/issues/filter.sorting.pagination.test.ts

```diff
@@ -95,7 +95,10 @@ describe(["@tier3"], "Filtering, sorting and pagination in Issues", function ()
     });
 
     it("All issues - Filtering issues by name", function () {
-        Analysis.analyzeAll(applicationsList[0]);
+        // As there are 12 applications with index from 0 to 11, starting from 6 - analysis for daytrader app is
```

Comment:
We can shorten the comment.
// Analyzing daytrader app for pagination test to generate issues more than 10.